src/main/java/noppes/npcs/scripted/entity/ScriptPlayer.java

```diff
@@ -120,6 +124,14 @@ public boolean hasReadDialog(int id){
 		PlayerDialogData data = PlayerDataController.instance.getPlayerData(player).dialogData;
 		return data.dialogsRead.contains(id);
 	}
+	
+	public void readDialog(int id) {
+		PlayerDataController.instance.getPlayerData(player).dialogData.dialogsRead.add(id);
+	}
+	
+	public void unreadDialog(int id) {
+		PlayerDataController.instance.getPlayerData(player).dialogData.dialogsRead.remove(id);
+	}
```

Comment:
This'll help people out a ton! I'm going to be moving these functions to a scripted object that just returns the entirety of a player's CNPC PlayerData. Something like:

var playerData = player.getData();

which would return a player's PlayerData object, then:

playerData.getDialogData()

which would return a PlayerDialogData object, where these functions go.

Reply:
I would like these functions to be left there for the sake of convenience, and I'd have to change some of my scripts if removed...but it makes sense for the sake of consistency.